Px.Utils.UnitTests/Px.Utils.UnitTests.csproj

```diff
@@ -20,4 +21,10 @@
     <ProjectReference Include="..\Px.Utils\Px.Utils.csproj" />
   </ItemGroup>
 
+  <ItemGroup>
```

Comment:
This should be removed. All file benchmarks can be done with the benchmark tool once its merged.